### ISSUE TITLE
AUR package

### ISSUE THREAD

**Issue by liara** [sharkdp/insect#199]
The AUR package fails to build. Any ideas?

```
> insect@5.1.0 postinstall /home/me/builds/insect/pkg/insect/usr/lib/node_modules/insect
> bower install && npm run browserify && npm run copy

evalmachine.<anonymous>:25
'use strict';
^

ReferenceError: internalBinding is not defined
    at evalmachine.<anonymous>:25:1
    at Object.<anonymous> (/usr/lib/node_modules/bower/node_modules/graceful-fs/fs.js:11:1)
    at Module._compile (internal/modules/cjs/loader.js:816:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:827:10)
    at Module.load (internal/modules/cjs/loader.js:685:32)
    at Function.Module._load (internal/modules/cjs/loader.js:620:12)
    at Module.require (internal/modules/cjs/loader.js:723:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at Object.<anonymous> (/usr/lib/node_modules/bower/node_modules/graceful-fs/graceful-fs.js:3:27)
    at Module._compile (internal/modules/cjs/loader.js:816:30)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! insect@5.1.0 postinstall: `bower install && npm run browserify && npm run copy`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the insect@5.1.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

https://aur.archlinux.org/packages/insect/#comment-697328

**Comment by sharkdp** [comment]
Thank you for reporting this. I saw the comment on AUR, but didn't have time to look into it so far.

Any help would be greatly appreciated.

**Comment by sharkdp** [comment]
I'm guessing that this is an issue with a newer version of npm/node. We probably have to update a few dependencies...

**Comment by liara** [comment]
Looks like it fails on graceful-fs. Maybe related to https://github.com/isaacs/node-graceful-fs/issues/151#issuecomment-483777776?

**Comment by cmcaine** [comment]
There are binaries available. Just package those instead?

**Comment by sharkdp** [comment]
This is hopefully fixed in 5.1.2: https://aur.archlinux.org/packages/insect/

> There are binaries available. Just package those instead?

I'd rather not. The binaries are huge and ship a full JavaScript engine with them.
